shared/rust/src/api/endpoints/image/recent.rs

```diff
@@ -54,13 +38,15 @@ impl ApiEndpoint for Create {
 /// * ['BadRequest'](http::StatusCode::BAD_REQUEST) if the request is malformed.
 pub struct Update;
```

Comment:
Should rename this (and relevant functions) to `Put` to reflect the change:
* does an "upsert", rather than an update to an existing resource